schema/omega-services.async.api.yaml

```diff
@@ -777,6 +830,199 @@ components:
       type: boolean
       description: true to indicate that this Agent is also considered an Authority File, false otherwise
 
+    RecordSummary:
+      type: object
+      required:
+        - identifier
+        - type
+        - creator
+        - current-description
+        - description
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        type:
+          $ref: '#/components/schemas/RecordType'
+        creator:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the Creating Body of the Record concept, i.e. a URI to an Agent
+        current-description:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the current description of the Record concept
+        description:
+          type: array
+          items:
+            $ref: '#/components/schemas/RecordDescriptionSummary'
+
+    RecordFull:
+      type: object
+      required:
+        - identifier
+        - type
+        - creator
+        - current-description
+        - description
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        type:
+          $ref: '#/components/schemas/RecordType'
+        creator:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the Creating Body of the Record concept, i.e. a URI to an Agent
+        current-description:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the current description of the Record concept
+        description:
+          type: array
```

Comment:
We're saying that `description` is an array - which makes sense as there can be more than one record description - however we are nowhere asking for `all` or `latest` so can I assume this will always be all of the descriptions?

Reply:
@rwalpole Yes, it will be all descriptions at present. This is the same as the Agent Service. It is only the `List`* services that take a `version-timestamp` (which defaults to `latest`) at present. I can see that it might also make sense to also extend this to the `GetAgent`* and `GetRecord`* services... would you like me to add that?

Reply:
In the current UI I think we have made the assumption that the user will always be editing the latest description. Therefore we can either let the UI work out which is the latest or make it part of the request. I would say it is better to handle this in the services.

Reply:
[PACT-1136](https://national-archives.atlassian.net/browse/PACT-1136) has been created to add an optional `version-timestamp` to the record request.